src/Contrib.KafkaFlow.Outbox.MongoDb/MongoDbOutboxRepository.cs

```diff
@@ -68,19 +86,56 @@ public async Task<IEnumerable<OutboxTableRow>> Read(int batchSize, CancellationT
         var documentIds = documents.Select(d => d.Id).ToList();
         var deleteFilter = Builders<OutboxDocument>.Filter.In(x => x.Id, documentIds);
 
-        await _collection.DeleteManyAsync(deleteFilter, token).ConfigureAwait(false);
+        await _collection.DeleteManyAsync(session, deleteFilter, cancellationToken: token).ConfigureAwait(false);
```

Comment:
We might need to delete LockAcquired from the collection once the operation completes successfully.

Reply:
I read about once the lock holder finishes processing a batch, the lock is automatically released, but where is it?

Reply:
The lock is not released automatically.
I thought about it, and decided to do a micro-optimisation: never release the lock.
The lock is just for 10 seconds, so in case of a crash, it'll take 10 seconds to recover.
The optimisation is to avoid that "release lock" call after every batch, but with batches, in practice, being very quick, it _could_ add up...